crates/api/examples/gcd.rs

```diff
@@ -38,8 +38,7 @@ fn main() -> Result<()> {
     let wasm = wat::parse_str(WAT)?;
 
     // Instantiate engine and store.
```

Comment:
Given that we're now using a default engine internally, should we amend this comment?